guides/doc-Managing_Configurations_Ansible/docinfo.xml

```diff
@@ -1,9 +1,9 @@
-<title>Configuring Red Hat Satellite to use Ansible</title>
+<title>Managing Configurations Using Ansible Integration in Red Hat Satellite</title>
 <productname>Red Hat Satellite</productname>
 <productnumber>6.12</productnumber>
-<subtitle>Configure Ansible in Satellite to help automate repetitive tasks</subtitle>
+<subtitle>Use Ansible integration in Satellite to configure host systems</subtitle>
 <abstract>
-    <para>This guide describes how to set up and configure Red Hat Satellite to use Ansible to perform remote execution and automate repetitive tasks.</para>
+    <para>This guide describes how to integrate Red Hat Satellite with Ansible and how to perform remote execution and automate repetitive tasks using Ansible roles and playbooks in Satellite.</para>
```

Comment:
I am unsure if foreman_ansible can actually run Ansible Playbooks. In docs, I've only found how to import playbooks into Foreman: https://docs.theforeman.org/nightly/Configuring_Ansible/index-katello.html#importing-an-ansible-playbook-by-name_ansible

Reply:
Ansible playbooks can be used in job templates and run in REX jobs, that's why we have a module on importing them. I think that we're not explicit enough about it in the REX chapter, though.
![image](https://user-images.githubusercontent.com/20555018/200808743-f4129063-a8ec-4252-a945-e581d507b706.png)